prov/efa/src/rdm/rxr_pkt_cmd.c

```diff
@@ -798,6 +798,7 @@ void rxr_pkt_handle_send_completion(struct rxr_ep *ep, struct rxr_pkt_entry *pkt
 		rxr_pkt_handle_eor_send_completion(ep, pkt_entry);
 		break;
 	case RXR_RMA_CONTEXT_PKT:
+		// how do we get an RMA Context Packet in a Send completion?
```

Comment:
RDMA-core did not return correct op code for read.

Reply:
Seems like if that's the only reason, we should let it fall through to the default.  I've left it in for now.